src/main/java/com/charlesgadeken/entwined/EntwinedGui.java

```diff
@@ -31,18 +53,127 @@ public void settings() {
 
     @Override
     public void setup() {
-
         LXStudio.Flags flags = new LXStudio.Flags(this);
         flags.resizable = true;
         flags.useGLPointCloud = false;
         flags.startMultiThreaded = true;
 
         Model model = Model.fromConfigs();
 
-        new LXStudio(this, flags, model);
+        lx = new LXStudio(this, flags, model);
         this.surface.setTitle(WINDOW_TITLE);
+
+        engineController = new EngineController(lx);
+
+        if (ConfigLoader.enableNFC){
+            configureNFC(lx);
+        }
+
+        if (ConfigLoader.enableOutputBigtree) {
+            // MRG TODO
+            // lx.addEffect(new TurnOffDeadPixelsEffect(lx));
+            configureExternalOutput();
+        }
+
+        System.out.println("setup() completed");
+    }
+
+
+    /* configureExternalOutput */
+
+    void configureExternalOutput() {
+        // Output stage
+        try {
+            output = new LXDatagramOutput(lx);
+            datagrams = new LXDatagram[model.ipMap.size()];
```

Comment:
So `LXDatagramOutput` seems to not exist any more. 
@bbulkow , what format are these outputs? I can't determine by looking through the old LX...

Reply:
`DDPDatagram` it seems. Can you give this a spin against the hardware? These API's changed a bit, and I want to validate the output looks "right"